src/Orchard.Cms.Web/Modules/Orchard.OpenId/Startup.cs

```diff
@@ -0,0 +1,130 @@
+﻿using System;
+using System.Diagnostics;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.DataProtection;
+using Microsoft.AspNetCore.Modules;
+using Microsoft.AspNetCore.Routing;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.IdentityModel.Tokens;
+using OpenIddict;
+using Orchard.Data.Migration;
+using Orchard.Environment.Navigation;
+using Orchard.Environment.Shell;
+using Orchard.OpenId.Drivers;
+using Orchard.OpenId.Indexes;
+using Orchard.OpenId.Models;
+using Orchard.OpenId.Recipes;
+using Orchard.OpenId.Services;
+using Orchard.OpenId.Settings;
+using Orchard.Recipes;
+using Orchard.Security.Permissions;
+using Orchard.Settings.Services;
+using YesSql.Core.Indexes;
+
+namespace Orchard.OpenId
+{
+    public class Startup : StartupBase
+    {
+        private readonly IDataProtectionProvider _dataProtectionProvider;
+        private readonly ILogger<Startup> _logger;
+
+        public Startup(
+            ShellSettings shellSettings,
+            IDataProtectionProvider dataProtectionProvider,
+            ILogger<Startup> logger)
+        {
+            _dataProtectionProvider = dataProtectionProvider.CreateProtector(shellSettings.Name);
+            _logger = logger;
+        }
+
+        public override void Configure(IApplicationBuilder builder, IRouteBuilder routes, IServiceProvider serviceProvider)
+        {
+            var openIdService = serviceProvider.GetService<IOpenIdService>();
+            var settings = openIdService.GetOpenIdSettingsAsync().GetAwaiter().GetResult();
+            if (!openIdService.IsValidOpenIdSettings(settings))
+            {
+                _logger.LogWarning("The OpenID module is not correctly configured.");
```

Comment:
In strings that are going to be shown to the user we are using OpenID Connect, (on next commit I have added "Connect" postfix to this string and another one that didn't use it.
All the other uses of OpenId or openId are vars, constants and namespaces like this AspNet.Security.OpenIdConnect.Extensions;